<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tvRemote</class>
 <widget class="QMainWindow" name="tvRemote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>680</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TV_Remote</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>680</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="1">
     <widget class="QLineEdit" name="Volume_display">
      <property name="font">
       <font>
        <pointsize>26</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="Btn_0">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Btn_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="Btn_vol_Down">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Vol-</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="Btn_ch_Down">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>CH-</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="Btn_8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="Btn_7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="Btn_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="Btn_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="Btn_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="Btn_ch_Up">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>CH+</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="Btn_vol_Up">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Vol+</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="Btn_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="Btn_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="Btn_9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="Channel">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Channel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="Volume">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Volume</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLineEdit" name="Channel_display">
      <property name="font">
       <font>
        <pointsize>26</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Btn_0</sender>
   <signal>clicked()</signal>
   <receiver>tvRemote</receiver>
   <slot>on_ButtonDigit_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_1</sender>
   <signal>clicked()</signal>
   <receiver>tvRemote</receiver>
   <slot>on_ButtonDigit_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_2</sender>
   <signal>clicked()</signal>
   <receiver>tvRemote</receiver>
   <slot>on_ButtonDigit_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_3</sender>
   <signal>clicked()</signal>
   <receiver>tvRemote</receiver>
   <slot>on_ButtonDigit_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_4</sender>
   <signal>clicked()</signal>
   <receiver>tvRemote</receiver>
   <slot>on_ButtonDigit_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_5</sender>
   <signal>clicked()</signal>
   <receiver>tvRemote</receiver>
   <slot>on_ButtonDigit_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_6</sender>
   <signal>clicked()</signal>
   <receiver>tvRemote</receiver>
   <slot>on_ButtonDigit_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_9</sender>
   <signal>clicked()</signal>
   <receiver>tvRemote</receiver>
   <slot>on_ButtonDigit_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_8</sender>
   <signal>clicked()</signal>
   <receiver>tvRemote</receiver>
   <slot>on_ButtonDigit_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_7</sender>
   <signal>clicked()</signal>
   <receiver>tvRemote</receiver>
   <slot>on_ButtonDigit_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_ButtonDigit_clicked()</slot>
 </slots>
</ui>
